nx-X11/programs/Xserver/hw/nxagent/man/nxagent.1

```diff
@@ -441,9 +441,22 @@ enable proxy binding mode
 .B \-options \fIfilename\fP
 path to an options file containing nx/nx options (see below).
 .PP
-Other than the command line options, \fBnxagent\fR can be configured at
-session startup and at runtime (i.e. when resuming a suspended session)
-by so-called nx/nx options.
+Other than the command line options, \fBnxagent\fR can be configured
+at session startup and at runtime (i.e. when resuming a suspended
```

Comment:
`i.e.,`

Reply:
I'd rather leave it that way. We are not uniform in this respect, there are occurances of both forms in the manpage and in the sourcecode.

https://jakubmarian.com/comma-after-i-e-and-e-g/ (I prefer British here)

Reply:
Okay, your choice (and likely makes sense since the statement is enclosed in parentheses.)